### PR TITLE
elpa: add cuda support, add libtool dep, and fix parallel build failure

### PR DESCRIPTION
**Pull request by haampie**

### CODE REVIEW

File: var/spack/repos/builtin/packages/elpa/package.py [hunk 1]
```diff
@@ -61,14 +62,11 @@ def headers(self):
         hlist.directories = [incdir]
         return hlist
 
-    build_directory = 'spack-build'
-
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/elpa/package.py]: Am I getting this wrong or are we now building in-source? Why was this necessary?

File: var/spack/repos/builtin/packages/elpa/package.py [hunk 2]
```diff
@@ -100,3 +109,7 @@ def configure_args(self):
         ])
 
         return options
+
+    def build(self, spec, prefix):
+        with working_dir(self.build_directory):
+            make(*self.build_targets, parallel=False)
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/elpa/package.py]: Why is this method overridden? The only difference I see from:

https://github.com/spack/spack/blob/1398038beead1a1532fc337165e2b6bf9d58c284/lib/spack/spack/build_systems/autotools.py#L324-L329

is `parallel=False`. Wouldn't an attribute at class level achieve the same effect?

**Review by alalazo — CHANGES_REQUESTED** [review]
A couple of minor questions and `flake8` to be fixed, but other than that LGTM. Would you like to sign up as maintainer of the package?

File: var/spack/repos/builtin/packages/elpa/package.py (hunk 1 above)
Comment (haampie) [reply var/spack/repos/builtin/packages/elpa/package.py]: Yeah, sorry, this was unintended

File: var/spack/repos/builtin/packages/elpa/package.py (hunk 2 above)
Comment (haampie) [reply var/spack/repos/builtin/packages/elpa/package.py]: Thanks, wasn't aware that was a thing

**Review by alalazo — APPROVED** [review]

**Comment by haampie** [comment]
Had to rebase it, so now it is actually done
